ChangeLog

```diff
@@ -1,3 +1,10 @@
+Changes in squid-6.5 (5 Nov 2023):
+
+	- Bug 5309: frequent "lowestOffset () <= target_offset" assertion
```

Comment:
We may also want to mention the removal of the assertion itself (when/if that [change](https://github.com/squidadm/squid/commit/ba1d41447868cd4fadb0d66e44f19e99a37b382c) gets merged as [planned](https://github.com/squid-cache/squid/pull/1563#issuecomment-1789175670)).